src/runtime/js/ts.ui/layout/layout-gui@tradeshift.com/spirits/asides/ts.ui.SideShowSpirit.js

```diff
@@ -361,6 +361,9 @@ ts.ui.SideShowSpirit = (function using(
 		 * @param {number} pct
 		 */
 		_position: function(pct) {
+			if (this.element.className.indexOf('ts-sidebar-first') > 0) {
```

Comment:
What's the reason for the class name? 🤔 why not `ts-sidebar-opposite` or something else?

Reply:
I don't know. It has been there for a long time. I just use it to check it from left or right side.